README.md

```diff
@@ -82,25 +136,18 @@ npx install-peerdeps eslint-plugin-nimbus-clean --yarn
       "plugins": [
         "nimbus-clean"
       ]
-    }
-    ```
+   }
+   ```
 
 2. If you don't have a `.prettierrc` config, please add it
 
-    ```prettier
-    {
-      "singleQuote": true,
-      "printWidth": 150,
-      "useTabs": false,
-      "tabWidth": 2,
-      "trailingComma": "all",
-      "semi": false
-    }
-    ```
-
-### License
```

Comment:
Please keep the License and Contributions section in this file

Reply:
@Vidhanvyrs Hi! Could you fix the last thing please?

And I'll merge it

Reply:
Ya sure